dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -127,12 +130,82 @@ const searchBar = css`
 	${from.desktop} {
 		display: none;
 	}
-	margin-left: 45px;
+	box-sizing: border-box;
+	display: block;
+	margin-left: 13px;
+	max-width: 23.75rem;
```

Comment:
Rems are not used in dcr, we favour pixels (1rem = 16pixels so this would be 380px)